### PR TITLE
Revision and edit of Compose install doc

### PR DESCRIPTION
**Pull request by fredlf**
Signed off by: Fred Lifton (fred.lifton@docker.com)

### CODE REVIEW

**Comment by bfirsh** [comment]
Needs signing off! :)

File: docs/install.md [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+to your `~/.bashrc` file.
+
+For complete instructions, or if you are on another platform, consult Docker's
+[installation instructions](https://docs.docker.com/installation/).
 
-There are also guides for [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) and [other platforms](https://docs.docker.com/installation/) in Docker’s documentation.
+### Install Compose
 
-Next, install Compose:
+To install Compose, run the following `curl` command:
```
Comment (aanand) [inline docs/install.md]: Actually [pushes bridge of glasses] this is a `curl` followed by a `chmod`. Perhaps we should stop obfuscating and split them onto two lines?

File: docs/install.md [hunk 2]
```diff
@@ -1,29 +1,44 @@
...
+[installation instructions](https://docs.docker.com/installation/).
 
-There are also guides for [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) and [other platforms](https://docs.docker.com/installation/) in Docker’s documentation.
+### Install Compose
 
-Next, install Compose:
+To install Compose, run the following commands:
 
-    curl -L https://github.com/docker/docker-compose/releases/download/1.1.0-rc1/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose; chmod +x /usr/local/bin/docker-compose
+    curl -L https://github.com/docker/docker-compose/releases/download/1.1.0-rc1/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```
Comment (aanand) [inline docs/install.md]: The version should now say `1.1.0-rc2`.

**Comment by aanand** [comment]
Minor typo and needs signoff; otherwise, LGTM.

**Comment by fredlf** [comment]
@aanand @bfirsh Okay, should be good to go now. Not sure who merges... guess we should figure that out!

**Comment by bfirsh** [comment]
@fredlf We follow the Docker process, roughly. Two LGTMs from a [maintainer](https://github.com/docker/fig/blob/master/MAINTAINERS).

Perhaps we should have additional process for docs?

**Comment by fredlf** [comment]
Well, the docs process currently is two maintainers, or just one if the PR originates from a docs maintainer. So.... for now:  ping @jamtur01 @SvenDowideit 

Two of us plus one technical (code) maintainer should be adequate, eh?

**Comment by jamtur01** [comment]
LGTM

**Comment by fredlf** [comment]
Thanks, @jamtur01 

@bfirsh You okay with me merging then?

**Comment by bfirsh** [comment]
Okay. Let's count you as a maintainer for docs on here, so that'll just require another set of eyes from a Fig maintainer.

Also: LGTM. Thanks. :)

**Pull request merged**